test/DebtQueue.t.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+import 'forge-std/Test.sol';
+import { DebtQueue } from '../src/libs/DebtQueue.sol';
+
+contract DebtQueueTest is Test {
+    using DebtQueue for DebtQueue.DebtDeque;
+
+    DebtQueue.DebtDeque public queue;
+
+    function test_popFront() public {
```

Comment:
if we're already adding sanity tests here, perhaps worth adding a test with multiple items in the queue, showing the pop indeed pops the first item added?

Reply:
added f44c74940f9acce59e5c81ab00f92d9d76a4373c